pkg/cmd/roachtest/roachtestflags/flags.go

```diff
@@ -359,8 +359,18 @@ var (
 
 	UseSpotVM bool
 	_         = registerRunFlag(&UseSpotVM, FlagInfo{
-		Name:  "use-spot",
-		Usage: `Use SpotVM to run tests, If the provider does not support spotVM, it will be ignored`,
+		Name: "use-spot",
+		Usage: `
+			May use SpotVM to run tests; note, this is merely a _hint_. The framework decides if a SpotVM should be used. 
+			You can override by passing --use-spot-always.`,
+	})
+
+	AlwaysUseSpotVM bool
+	_               = registerRunFlag(&UseSpotVM, FlagInfo{
```

Comment:
`&AlwaysUseSpotVM` instead 😄

Reply:
Oops, that would have been frustrating to someone flooring the gas pedal :)